modules/ROOT/pages/configuration/plugins.adoc

```diff
@@ -0,0 +1,59 @@
+[[plugins]]
+= Plugins
+:description: This page describes how to load plugins to a Neo4j deployment.
+
+This page describes how to load plugins to a Neo4j deployment.
```

Comment:
I might want to add here some intro about the plugins as this would be the main page in the manual about plugins. We should mention what they are and how they are meant to be used, and that some of them require activation key.